roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -0,0 +1,61 @@
+# This file is included both in the openshift_master role and in the upgrade
+# playbooks.  For that reason the ha_svc variables are use set_fact instead of
+# the vars directory on the role.
+
+- debug: var=openshift_version
+
+- name: Init HA Service Info
+  set_fact:
+    ha_svc_svc_dir: "/usr/lib/systemd/system"
```

Comment:
Thinking about this a bit, we may want to just go with /etc/systemd/system instead of /usr/lib/systemd/system, since /usr/lib/systemd/system should only contain units related to installed packages.

Of course, that would require a migration for existing installs, but the 3.2 upgrade is probably the best time to do that :)